parcel_test.go

```diff
@@ -31,57 +32,93 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db") // настройте подключение к БД
+	require.NoError(t, err)
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
 	// add
 	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
+	p, err := store.Add(parcel)
+	require.NoError(t, err)
+	require.NotNil(t, p)
 	// get
 	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
-
+	pers, err := store.Get(p)
+	require.NoError(t, err)
+	assert.Equal(t, parcel.Client, pers.Client)
+	assert.Equal(t, parcel.Status, pers.Status)
+	assert.Equal(t, parcel.Address, pers.Address)
+	assert.Equal(t, parcel.CreatedAt, pers.CreatedAt)
 	// delete
 	// удалите добавленную посылку, убедитесь в отсутствии ошибки
 	// проверьте, что посылку больше нельзя получить из БД
+	err = store.Delete(p)
+	require.NoError(t, err)
+	pers, err = store.Get(p)
+	require.Error(t, err)
+	require.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```